apps/api/src/modules/summary/entities/summary.entity.ts

```diff
@@ -20,17 +19,16 @@ export class Summary {
   @IsOptional()
   inputFile: string;
 
-  @Column({ name: 'output_text' })
-  @Field()
-  @IsOptional()
-  outputFile: string;
+  @Column({ name: 'output_text', nullable: true })
+  @Field({ nullable: true })
+  outputText: string;
 
   @CreateDateColumn({ name: 'created_on' })
   @Field()
   createdOn: Date;
 
-  @UpdateDateColumn({ name: 'modified_on' })
-  @Field()
+  @UpdateDateColumn({ name: 'modified_on'})
+  @Field({ nullable: true })
```

Comment:
This wont be nullable. If you are making any changes compared to this, chances are you are making some mistake

https://github.com/OsmosysSoftware/osmo-x/blob/d854a63f0429bc72d850796e9c39ddff23d7622d/apps/api/src/modules/notifications/entities/notification.entity.ts#L55

Reply:
Okay reverted the changes, this is working fine as of now